build/conf.js

```diff
@@ -21,6 +21,7 @@ import path from 'path';
  * Load the i18n and l10n configuration. Used when dashboard is built in production.
  */
 let localization = require('../i18n/locale_conf.json');
+let util = require('gulp-util');
```

Comment:
Sorry if I haven't clearly explained it. Look few lines up at line 18. `import path from 'path'`. I've wanted to move it there to common imports and use this syntax :)